test/02-ctr-logs.bats

```diff
@@ -74,4 +75,112 @@ run_conmon_with_log_opts() {
     run_conmon_with_log_opts --log-path "k8s-file:$LOG_PATH" --log-path "$invalid_log_driver:$LOG_PATH"
     assert_failure
     assert_output_contains "No such log driver $invalid_log_driver"
-}
\ No newline at end of file
+}
+
+@test "ctr logs: journald with --log-label, no '=' in label" {
+    start_conmon_with_default_args \
+        --log-path "journald:" \
+        --log-label "CONMON_TEST_LABEL1"
+
+    assert_output_contains "Container labels must be in format LABEL=VALUE"
+}
+
+@test "ctr logs: journald with --log-label, multiple '=' in label" {
+    start_conmon_with_default_args \
+        --log-path "journald:" \
+        --log-label "CONMON_TEST_LABEL1=FOO=$CTR_ID"
+
+    assert_output_contains "Container labels must be in format LABEL=VALUE"
+}
+
+@test "ctr logs: journald with --log-label, no label name" {
+    start_conmon_with_default_args \
+        --log-path "journald:" \
+        --log-label "=$CTR_ID"
+
+    assert_output_contains "Container labels must be in format LABEL=VALUE"
+}
+
+@test "ctr logs: journald with --log-label, invalid character" {
+    start_conmon_with_default_args \
+        --log-path "journald:" \
+        --log-label "MY%LABEL=$CTR_ID"
+
+    assert_output_contains "Container label names must contain only uppercase letters, numbers and underscore"
+}
+
+@test "ctr logs: k8s-file with --log-label" {
+    start_conmon_with_default_args \
+        --log-path "k8s-file:$LOG_PATH" \
+        --log-label "CONMON_TEST_LABEL1=$CTR_ID"
+
+    assert_output_contains "k8s-file doesn't support --log-label"
+}
+
+@test "ctr logs: journald with --log-label" {
+    run_conmon_with_default_args \
+        --log-path "journald:" \
+        --log-label "CONMON_TEST_LABEL1=$CTR_ID" \
+        --log-label "CONMON_TEST_LABEL2=$CTR_ID"
+
+    run journalctl --user CONMON_TEST_LABEL1="$CTR_ID"
+    assert "${output}" =~ "hello from busybox"
+
+    run journalctl --user CONMON_TEST_LABEL2="$CTR_ID"
+    assert "${output}" =~ "hello from busybox"
+
+    run journalctl --user NON_EXISTING_LABEL="$CTR_ID"
+    assert "${output}" !~ "hello from busybox"
+}
+
+@test "ctr logs: k8s-file with --log-tag" {
+    start_conmon_with_default_args \
+        --log-path "k8s-file:$LOG_PATH" \
+        --log-tag "CONMON_TEST_LABEL1"
+
+    assert_output_contains "k8s-file doesn't support --log-tag"
+}
+
+@test "ctr logs: journald with --log-tag" {
+    run_conmon_with_default_args \
+        --log-path "journald:" \
+        --log-tag "tag_$CTR_ID"
+
+    run journalctl --user CONTAINER_TAG="tag_$CTR_ID"
+    assert "${output}" =~ "hello from busybox"
+}
+
+@test "ctr logs: journald priority level" {
```

Comment:
Isn't this a duplicate of what we test in `06-priority-parsing.bats`?

Reply:
True. I've removed that test.